test/integration/encode_decode_cycle_test.go

```diff
@@ -69,15 +67,13 @@ func TestEncodeDecodeEncodeCycleComparison(t *testing.T) {
 				format := syft.FormatByID(test.formatOption)
 				require.NotNil(t, format)
 
-				by1, err := syft.Encode(originalSBOM, format)
-				assert.NoError(t, err)
+				by1 := encode(t, originalSBOM, format)
```

Comment:
I'm also a little confused about the diff here. I don't see changes to `syft.Encode` and thought that this test was specifically for checking if changes to that method cause lossy behavior across conversions. Why have we updated this to a custom test function that now uses the `format.Encode` method?

Reply:
outdated, please ignore